oat-sa/extension-tao-testqti
# Feature/ADF-1752/Confirm category propagation

### Related to: https://oat-sa.atlassian.net/browse/ADF-1752

Requires: 
 - [x] https://github.com/oat-sa/tao-core/pull/4079 for the style fix when using the Solar UI

### Summary

Allow the user to propagate partial categories to contained items.

### Details

When a tag applies to a part only of the contained items, the user can click on it to propagate it to the contained items. A confirmation dialog is presented before so the user can cancel or validate.

### How to test

- checkout the branch: `git checkout -t origin/feature/ADF-1752/confirm-category-propagation`
- open TAO backoffice, then author a test with multiple items in it
- set a custom category to a test part or a section
- from an item, remove the category
- go back to the test part or section and click the partial tag
- a confirmation dialog should pop up
- clicking ok will apply the change, clicking cancel should do nothing
